packages/experiments/src/theming/examples/Themes.tsx

```diff
@@ -149,14 +152,29 @@ export const invertedDefaultTheme: ITheme = createTheme({
 });
 
 export const invertedPrimaryTheme: ITheme = createTheme({
-  // TODO: Components need to be converted to use semanticColors instead of palette.
-  //        For now, we need to invert palette to style components like: CommandBar.
-  //        Comment this line out to see the effects.
   palette: invertedPrimaryPalette,
   semanticColors: {
     bodyText: defaultTheme.palette.white,
     bodyBackground: defaultTheme.palette.themePrimary
   }
 });
 
-export const schemeTheme: ITheme = createTheme({});
+export const schemeThemeVariants: ITheme = {
+  ...defaultTheme,
+  schemes: {
```

Comment:
so right now for testing purposes, I understand you're just manually defining schemes. Are we intending to automatically create them in the future?

Reply:
ah, nm, I see you have a comment addressing that in the PR description
my 2 cents: all this effort is to make it effortless to apply a scheme to a context, so we should add it by default when creating/loading a theme

Reply:
If we do create default schemes, David and I are in agreement that we should fold variants functions into styling as opposed to creating a circular dependency between styling and variants packages (per our existing email thread)

Reply:
I think it will impact bundle size to do this. But I'm not sure how badly. It would very much be nice to minimize the impact, but it would be very very nice to have these schemes readily available by simply adding the right prop to a component.